assets/css/_shims.scss

```diff
@@ -20,12 +20,10 @@
   }
 }
 
-@include media($medium-screen) {
-  .app-header {
-    .tbds-button.app-header__announce-button {
-      .tbds-button__icon--start {
-        margin-inline-end: 0;
-      }
+.app-header__announce-button {
+  .tbds-button__icon--start {
+    @media (max-width: $breakpoint-2) {
```

Comment:
Did we want to keep this as `max-width` for any reason?

Reply:
Yeah, this is a specific case. It’s in the `_shims.scss` partial and is fixing a regression the design system is causing. All styles in this partial should be temporary fixes and be removed once the real fix is in place.

Basically, we need to rethink Constable's “announce” button on small screens, or the design system needs to allow icon-only buttons at specific viewports.